promptsource/templates/squad_v2/templates.yaml

```diff
@@ -60,9 +61,11 @@ templates:
     metadata: !TemplateMetadata
```

Comment:
Can you make this one more clear, something like "Determine the question that you might ask to the the following answer for the given context."